resources/js/store/modules/user/mutations.js

```diff
@@ -119,6 +119,14 @@ export const mutations = {
         state.user[payload.column] = payload.v;
     },
 
+    /**
+     * The user wants to change a tagging setting
+     */
+    tagging (state, payload)
+    {
+        state.user[payload.column] = payload.v;
```

Comment:
This is a dynamic function that is being repeated. We should either use 1 dynamic function and pass whatever keys we want to update, or else have static functions that only change a specific param